pom.xml

```diff
@@ -26,7 +26,7 @@
     </parent>
 
     <artifactId>chronicle-bytes</artifactId>
-    <version>2.22ea11-SNAPSHOT</version>
+    <version>2.22ea12-SNAPSHOT</version>
```

Comment:
why is this changed?

Reply:
`develop` moved on and I changed it when reproducing errors in Wire. I should have reverted it back before I created the PR.